tests/cupyx_tests/fallback_mode_tests/test_fallback.py

```diff
@@ -147,3 +149,101 @@ def test_func_not_in_numpy(self):
 
         with self.assertRaises(AttributeError):
             func = fallback_mode.numpy.dummy  # NOQA
+
```

Comment:
Add `test_ndarray` which checks `numpy.ndarray(shape)`.

Reply:
Done

Reply:
Hmm, `fallback_mode.numpy.ndarray(shape)` seems to work well because `fallback_mode.numpy.ndarray` is of `_RecursiveAttr` type. I'm wondering `isinstance(array, numpy.ndarray)` is not work well.

Reply:
Yeah, I get it. No, currently that won't work. You need `fallback_mode.numpy.ndarray` to point towards `utils.ndarrray`.
But, it would be like having two `__init__` methods for `utils.ndarray`.

Reply:
`utils` should not be exposed to users because the original `numpy` have no attributes `utils`.

Reply:
Will do it in next PR as discussed